utils/cronjobs_osgeo_lxd/cron_grass_current_stable_build_binaries.sh

```diff
@@ -80,6 +77,31 @@ halt_on_error()
 # function to configure for compilation
 configure_grass()
 {
+# setup source code repo
+
+mkdir -p $SOURCE $TARGETDIR
+
+# fetch repo if needed
+cd "$SOURCE/"
+# Check if the repository is already cloned
+if [ -d "$BRANCH" ]; then
+  echo "The GRASS GIS repository <$BRANCH> has already been cloned. Continuing..."
+else
+  echo "Cloning the GRASS GIS repository <$BRANCH> first..."
+  git clone https://github.com/OSGeo/grass.git $BRANCH
+  if [ $? -eq 0 ]; then
+    echo "Repository successfully cloned."
+  else
+    echo "Error: Failed to clone the repository."
+    exit 1
+  fi
+fi
```

Comment:
I don't understand what this is here for. I already had to get grass and grass-addons repos just to get the script running. Maybe this is needed too, but that's not clear to me right now from the setup instructions.

Reply:
The idea was to automatically check out the code. Of course, a lot of lines for little gain, since this should be done beforehand. But then the script is no longer standalone.

Question: Is it safe to assume that the repo has already been cloned? If so, then I can happily shrink this part.

Reply:
My point is that the script and your existing instructions (apt and the /home/src dir) required to have the grass repo downloaded beforehand, so there is no point in downloading it in the script. (Unless your idea was to remove the requirement.) The current instructions look good, so that seems to me like removing the code here.

Reply:
Agreed. I have removed this part in 2477630cdc33117957239dd249f304e45868c818

Reply:
Also dropped "fetch addon repo if needed" in 6514132e185fa3685c79c78af6622de166e4d67a

See `README.md` for the (updated) docker usage.